app/uk/gov/hmrc/minorentityidentificationfrontend/views/templates/govuk_template.scala.html

```diff
@@ -30,6 +30,17 @@
 
 @(pageConfig: PageConfig, title: String, startUrl: Option[String] = None)(contentBlock: Html)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
+@head = {
+<!--[if lte IE 8]>
+<link href=' @controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /> <![endif]-->
+<!--[if gt IE 8]><!-->
+<link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" />
+<!--<![endif]-->
```

Comment:
I think these need to be removed to fix the footer/autocomplete issue (hopefully the explanation makes sense in slack) but the timeout dialog and the accessibile-autocomplete.css need to stay